### PR TITLE
53447 components

### PR DESCRIPTION
**Pull request by Hornshade**
Changes after the login implementation

### CODE REVIEW

File: src/app/components/header/header.component.ts [hunk 1]
```diff
@@ -1,12 +1,49 @@
...
+		// 	this.user = data;
+		// });
+		this.loginService.currentUser.subscribe((data) => {
+			if (data !== null) this.user = data;
+		});
+	}
+
+	logout() {
+		localStorage.removeItem('userToken');
+		localStorage.removeItem('userId');
```
Comment (BiancaDianaAlexiuc) [inline src/app/components/header/header.component.ts]: You can use localStorage.clear to remove all from the local storage

File: src/app/components/list/list.component.scss [hunk 2]
```diff
@@ -1,73 +1,156 @@
 @import '../../../styles/variables.scss';
 
 // la content, nu ii pot da un height % ca sa-l fac responsive. am fortat cu min si max height in px
+// .c-list {
+//   display: flex;
```
Comment (BiancaDianaAlexiuc) [inline src/app/components/list/list.component.scss]: Do we need this commented code?

**Review by BiancaDianaAlexiuc — APPROVED** [review]

File: src/app/components/header/header.component.ts (hunk 1 above)
Comment (Hornshade) [reply src/app/components/header/header.component.ts]: Thanks. I'll use that instead.

File: src/app/components/list/list.component.scss (hunk 2 above)
Comment (Hornshade) [reply src/app/components/list/list.component.scss]: Thanks. I removed it now. It was css from before changing the list component to be different based on users that I forgot to remove.

**Pull request merged**